test/create-prompt.test.ts

```diff
@@ -0,0 +1,38 @@
+import { describe, it, expect } from 'vitest';
+import { FileStorageProvider } from '../src/storage/fileStorage';
+import { createPrompt } from '../src/models/prompt';
+
+describe('FileStorageProvider - Create Prompt', () => {
+  it('creates and saves a new prompt', async () => {
+    const storage = new FileStorageProvider();
+    await storage.initialize();
+
+    const prompt = createPrompt('Test Title', 'Test Content', 'General', ['tag1']);
+    await storage.save(prompt);
+
+    const prompts = await storage.loadAllPrompts();
+    expect(prompts).toHaveLength(1);
+    expect(prompts[0].title).toBe('Test Title');
+    expect(prompts[0].content).toBe('Test Content');
+    expect(prompts[0].category).toBe('General');
+    expect(prompts[0].tags).toEqual(['tag1']);
```

Comment:
I'm probably going to remove tag support. I think title / description / category should be sufficient.